lib/dlss/capistrano/tasks/sidekiq.rake

```diff
@@ -0,0 +1,51 @@
+# Capistrano plugin hook to set default values
+namespace :load do
+  task :defaults do
+    set :sidekiq_role, fetch(:sidekiq_role, :app)
+  end
+end
+
+# Integrate sidekiq hooks into Capistrano
+namespace :deploy do
+  before :starting, :add_sidekiq_hooks do
+    invoke 'sidekiq:add_hooks'
+  end
+end
+
+namespace :sidekiq do
+  # NOTE: no `desc` here to avoid publishing this task in the `cap -T` list
+  task :add_hooks do
```

Comment:
Is it possible to have this opt in (or out at least)? I could see it possible of consuming both this version of dlss-capistrano and capistrano-sidekiq.

Reply:
Sure, I'll make it opt-in.

Reply:
done! have a look-see.